tests.py

```diff
@@ -11,6 +11,8 @@
     Link,
     Query,
     Template,
+    ArrayProperty,
```

Comment:
Please resort these imports. The convention I use for this project (although not your fault as I haven't added this to the documentation yet) is to sort the imports asciibetically.